gcemetadata/gcemetadata

```diff
@@ -180,6 +178,14 @@ def main():
                 print >> sys.stderr, '--identity argument must have a value'
                 sys.exit(1)
             meta.set_identity_arg(val)
+        elif opt == '--identity-format':
+            supported_vals = ['full', 'standard']
+            if val not in supported_vals:
+                msg = '--identity-format possible values: "%s"' %supported_vals
+                print >> sys.stderr, msg
+                sys.exit(1)
+            meta.set_identity_format(val)
+            continue
```

Comment:
No the continue is necessary of the --identity-format which is not an endpoint url will be added to the options passed for query and thus "unavailable" will be printed.